packages/analytics/src/factory.ts

```diff
@@ -226,7 +229,11 @@ export function factory(
     app,
     // Public methods return void for API simplicity and to better match gtag,
     // while internal implementations return promises.
-    logEvent: (eventName, eventParams, options) => {
+    logEvent: (
+      eventName: string,
```

Comment:
Is it necessary to type them explicitly? Are they not inferred by `FirebaseAnalyticsInternal ` ?

Reply:
It used to be fine but after adding the logEvent overloads to analytics-types now it complains if the params aren't typed?